### PR TITLE
refactor: Change to use submodule

### PR DESCRIPTION
**Pull request by richtera**
Add sample build

### CODE REVIEW

**Comment by richtera** [comment]
Ok the zip artifacts are now clean and have the version number in and outside of the zip file.

File: .github/workflows/update-cli.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Update submodules
```
Comment (frozeman) [inline .github/workflows/update-cli.yaml:1]: lets not do that automatically

File: src/scripts/bundle_proxy_win.bat [hunk 2]
```diff
@@ -25,10 +25,10 @@ mkdir %DISTWORDSPATH% > nul 2> nul
 mkdir %TARGETPACKAGESPATH% > nul 2> nul
 
 rem Getting all the requirements
-python -m pip install -r %ETH2REQUIREMENTSPATH% --target %TARGETPACKAGESPATH%
+python -m pip install -r %ETH2REQUIREMENTSPADISTBINPATHTH% --target %TARGETPACKAGESPATH%
```
Comment (frozeman) [inline src/scripts/bundle_proxy_win.bat]: why such a long name `ETH2REQUIREMENTSPADISTBINPATHTH` ?

Comment (richtera) [reply src/scripts/bundle_proxy_win.bat]: I just copied what they had. Someone wanted to make this cryptic :) haha.

Comment (richtera) [reply src/scripts/bundle_proxy_win.bat]: I will check the original and minimize my changes to make unnecessary things drop out.

Comment (richtera) [reply src/scripts/bundle_proxy_win.bat]: Fixed.

**Review by frozeman — CHANGES_REQUESTED** [review]
lets remove the autoupdate of the submodule please

File: .github/workflows/update-cli.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+# Controls when the action will run.
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```
Comment (frozeman) [inline .github/workflows/update-cli.yaml:6]: Is this it?

Comment (richtera) [reply .github/workflows/update-cli.yaml:6]: This means this workflow can be manually or triggered from the other repo using curl with the github token.

**Pull request merged**